src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -173,16 +173,16 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id,
 
   if (debug_logging_enabled) {
     Debug("ModuleLoadFinished: ", module_id, " ", module_info.assembly.name,
-            " AppDomain ", module_info.assembly.app_domain_id, " ",
-            module_info.assembly.app_domain_name);
+          " AppDomain ", module_info.assembly.app_domain_id, " ",
+          module_info.assembly.app_domain_name);
   }
 
   AppDomainID app_domain_id = module_info.assembly.app_domain_id;
 
   // Identify the AppDomain ID of mscorlib which will be the Shared Domain
   // because mscorlib is always a domain-neutral assembly
   if (!corlib_module_loaded && (module_info.assembly.name == "mscorlib"_W ||
```

Comment:
I'm worried that this shared domain logic will cause us to lose traces once merged so I think we'll need to modify one of our samples to be a "MultiDomainHost" and count Ado.NET traces. But this should be a followup PR because this PR is already large enough.